README.rst

```diff
@@ -166,6 +172,10 @@ Acquire sources, and install package in development mode::
     source .venv/bin/activate
     pip install --editable='.[testing]'
 
+Start a sandbox instance of MongoDB in another terminal::
+
+    docker run -it --rm --publish=27017:27017 mongo:5
```

Comment:
this PR adds support up to `4`, but you're using mongo 5 here?

Reply:
Probably a hiccup while splitting GH-6 vs. GH-8.

Reply:
I'd change this to 4, because testing with 5 should only come with the other PR, it will be confusing if someone checks the git log in the future, to see a PR that adds support for 2 & 3 and tests (also for existing 4), and see that 5 is used.

Reply:
Thanks. I've amended the most recent commit accordingly, see ccefb97564aa.